src/panels/lovelace/editor/config-elements/hui-stack-card-editor.ts

```diff
@@ -0,0 +1,237 @@
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  customElement,
+  property,
+  CSSResult,
+  css,
+} from "lit-element";
+import { classMap } from "lit-html/directives/class-map";
+
+import { struct } from "../../common/structs/struct";
+import { HomeAssistant } from "../../../../types";
+import { LovelaceCardEditor } from "../../types";
+import { StackCardConfig } from "../../cards/types";
+import { fireEvent } from "../../../../common/dom/fire_event";
+
+const cardConfigStruct = struct({
+  type: "string",
+  cards: ["any"],
+  title: "string?",
+});
+
+@customElement("hui-stack-card-editor")
+export class HuiAlarmPanelCardEditor extends LitElement
+  implements LovelaceCardEditor {
+  @property() public hass?: HomeAssistant;
+  @property() private _config?: StackCardConfig;
+  @property() private _selectedCard: number = 0;
+
+  public setConfig(config: StackCardConfig): void {
+    config = cardConfigStruct(config);
+    this._config = config;
+  }
+
+  protected render(): TemplateResult {
+    if (!this.hass || !this._config) {
+      return html``;
+    }
+    const selected = this._selectedCard!;
+    const numcards = this._config.cards.length;
+    const start = Math.max(0, Math.min(numcards - 5, selected - 2));
```

Comment:
Can we make the number of cards shown depend on screen width? Or maybe just use `paper-tabs` as it will handle it all for you.

Reply:
`paper-tabs` works, though that leaves a gap to the left and right even when the chevrons are hidden, but I guess that's ok.

Why doesn't `/config/` use that?

Reply:
Because it navigates to different pages and that doesn't work well.